### PR TITLE
Update DG on find, UG & PPP

### PR DESCRIPTION
**Pull request by kevinkuo0320**
This PR updates some documentation: 
- DG for find sequence diagram
- DG description for find 
- UG for import part
- PPP for kevinkuo0320

### CODE REVIEW

File: docs/diagrams/FindSequenceDiagram.puml [hunk 1]
```diff
@@ -34,11 +31,11 @@ deactivate FindCommandParser
 AddressBookParser -> FindCommandParser : parse("XYZ")
 activate FindCommandParser
 
-create Person
-FindCommandParser --> Person
-activate Person
-Person --> FindCommandParser : person
-deactivate Person
+create NameContainsKeywordsPredicate
+FindCommandParser --> NameContainsKeywordsPredicate
```
Comment (cwq2326) [inline docs/diagrams/FindSequenceDiagram.puml:35]: To be consistent, you can add in the constructor call when creating an object in the sequence diagram.

```suggestion
FindCommandParser --> NameContainsKeywordsPredicate : NameContainsKeywordsPredicate(arguments here)
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -425,39 +425,49 @@ Additionally, there is a static final static message to be displayed to the user
...
 ### Find feature
 
-#### Current Implementation
+The find mechanism implements the following sequence for the method call execute("find").
 
-The activity diagram below illustrates the flow of a `find` command.
+#### What is the find feature
 
-![FindFeatureActivityDiagram](images/FindFeatureActivityDiagram.png)
+The find feature allows users to find a particular contact in Tracey and retrieve their specific details.
```
Comment (cwq2326) [inline docs/DeveloperGuide.md:432]: Can add in find a person using their name

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -425,39 +425,49 @@ Additionally, there is a static final static message to be displayed to the user
...
+
+The `find` command is as follows:
+
+`find NAME`
 
-#### Usage Scenario
+The user do not require to search the full name of the contact. However, the name must be its substring.
```
Comment (cwq2326) [inline docs/DeveloperGuide.md:438]: Can add the query substring must match the person's name starting from the first character eg. `kev` will match `kevin` but `vin` will not

**Review by cwq2326 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 4]
```diff
@@ -43,15 +42,15 @@ We hope you enjoy Tracey as much as we did when developing it!
...
-|:-------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------:|
-|   :information_source:  |      <div markdown="span" class="alert alert-info"> Additional notes are parked here. They contain beneficial information in case you are confused.</div>      |
-|   ⚠️    |         <div markdown="span" class="alert alert-warning"> Warnings are parked here. They contain information that you **NEED** to take note of.</div>          |
-|   💡    | <div markdown="span" class="alert alert-primary"> Tips are parked here. They  highlight certain information that will allow you to fully utilise Tracey.</div> |
+| Symbol  |                                                 Purpose                                                 |
+|:-------:|:-------------------------------------------------------------------------------------------------------:|
+|   📓    |     Additional notes are parked here. They contain beneficial information in case you are confused.     |
```
Comment (cwq2326) [inline docs/UserGuide.md:51]: We have updated the icon for the note, so we are using ℹ️ (information_source)

**Review by cwq2326 — CHANGES_REQUESTED** [review]

**Review by cwq2326 — APPROVED** [review]
LGTM 💯

File: docs/diagrams/FindSequenceDiagram.puml [hunk 5]
```diff
@@ -7,21 +7,18 @@ participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
...
 activate LogicManager
-note RIGHT
-=XYZ = Alex
-end note
 
-LogicManager -> AddressBookParser : parseCommand("add XYZ")
+LogicManager -> AddressBookParser : parseCommand("find XYZ")
 activate AddressBookParser
 
 create FindCommandParser
```
Comment (jiale-c) [inline docs/diagrams/FindSequenceDiagram.puml:24]: Do add a constructor here.

**Review by jiale-c — CHANGES_REQUESTED** [review]
All good with one change needed.

**Pull request closed without merging**